src/components/Cart.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import './../css/cart.css';
+
+interface Product {
+    id: string;
+    image: string;
+    title: string;
+    price: number;
+    category: string;
+    description: string;
+    quantity: number;
+}
+
+interface CartProps {
+    cartProducts: Product[];
+    onClose: () => void;
+    onRemoveProduct: (product: Product) => void;
+}
+
+function ProductCard({ product, onRemoveProduct }: { product: Product; onRemoveProduct: (product: Product) => void }) {
+
+    const handleRemoveProduct = () => {
+        onRemoveProduct(product);
+    }
+
+    return (
+        <div className="cart-product-card">
+            <img src={product.image} alt={product.title} />
+            <h3>{product.title}</h3>
+            <p>Price: ${product.price}</p>
+            <p>{product.quantity}</p>
+            <button onClick={handleRemoveProduct}>Remove</button>
```

Comment:
No need to make this change, just a comment that our style on ADDI is to define function props inline unless they need to be memoized or are used in multiple places. Just minimizes the # of variable declarations on larger components.
```suggestion
            <button onClick={() => onRemoveProduct(product)}>Remove</button>
```